script/add-raml

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -eou pipefail
+
+ROOT=$(cd $(dirname $0)/..; pwd)
+source ${ROOT}/env
+source ${ROOT}/script/include/setup.sh
+
+STAGING=
+
+if [ ${1:-} = "--stage" ] || [ ${1:-} = "-s" ]; then
+  STAGING=yes
+  CONTENT_URL=${STAGING_CONTENT_URL}
+  shift
+else
+  CONTENT_URL=${PROD_CONTENT_URL}
+fi
+
+if [ -z "${1:-}" ]; then
+  cat <<EOM 1>&2
+Usage: script/add-raml [--stage|-s] <path>
+
+ --stage|-s - Submit content to the staging endpoint.
+ path - Path to a Raml file.
+
+EOM
+  exit 1
+fi
+
+# Acquire an API key.
+apikey raml ${STAGING}
+
+# If a custom CONTENT_ID_BASE is set in the environment, use it. Otherwise, read it from your
+# _deconst.json file.
+if [ -z "${CONTENT_ID_BASE:-}" ]; then  
+  if  [ "$PYVER" -eq "3" ]; then
+    CONTENT_ID_BASE=$(python -c "import json; print(json.loads(open('${1}/_deconst.json').read())['contentIDBase'])")
+  else
+    CONTENT_ID_BASE=$(python -c "import json; print json.load(open('${1}/_deconst.json'))['contentIDBase']")
+  fi
+fi
+
+# Derive CONTAINER_ROOT.
+if [ -z "${CONTAINER_ROOT:-}" ]; then
+  CONTAINER_ROOT="$(cd ${1} && git rev-parse --show-toplevel)"
+fi
+
+# Add visibility to endpoints being called
+echo "CONTENT_ID_BASE | ${CONTENT_ID_BASE}"
+echo "CONTENT_ROOT | ${1}"
+echo "CONTAINER_ROOT | ${CONTAINER_ROOT}"
+echo "CONTENT_URL | ${CONTENT_URL}"
+echo "COMPOSE_NETWORK_NAME | ${COMPOSE_NETWORK_NAME}"
+
+# Run the Jekyll builder (from a Docker container) on the provided content repository.
+docker run \
+  --rm=true \
+  -e ENVELOPE_DIR=${CONTAINER_ROOT}/_site/deconst-envelopes \
+  -e ASSET_DIR=${CONTAINER_ROOT}/_site/deconst-assets \
+  -e CONTENT_ID_BASE=${CONTENT_ID_BASE} \
+  -e VERBOSE=${VERBOSE:-} \
+  -e CONTENT_ROOT=${1} \
+  -v ${CONTAINER_ROOT}:${CONTAINER_ROOT} \
+  # raml-preparer:dev
```

Comment:
As written this commented line breaks the command (line does not continue). Please remove or move to the end of the statement.